install/kubernetes/helm/istio/requirements.yaml

```diff
@@ -46,3 +46,6 @@ dependencies:
     version: 0.8.0
     # repository: file://../galley
     condition: galley.enabled
+  - name: kiali
```

Comment:
Thoughts on having `kiali` be the "top level" chart which requires `istio`, instead of `istio` being the top level chart and relying upon `kiali`?

Perhaps a change that could be made down the line if `kiali` was hosted in `kubernetes/charts`.